spec/workers/twitter_worker_spec.rb

```diff
@@ -188,4 +188,35 @@
       expect(Tweet.first.sent_to_fb).to eq false
     end
   end
+
+  context 'when tweet is successfully saved to db' do
+    it 'broadcast infromation via proper action cabel channel' do
```

Comment:
`broadcast` -> `broadcasts`, replate `proper` by the channel name - avoid using `proper` or similar expressions in test - name or omit it